src/utils.ts

```diff
@@ -4,6 +4,7 @@ import { tmpdir } from 'os';
 import { existsSync, readFileSync, unlinkSync, writeFileSync } from 'fs';
 import { exec, execSync } from 'child_process';
 import { GitExtension, Repository } from './@types/git';
+import * as path from 'path'
```

Comment:
All look good. Could you put this on the second line?

Reply:
👌 Done.